### DISCUSSION TITLE
Openfast operational questions

### DISCUSSION THREAD

**Discussion by HiHabi** [OpenFAST/openfast#1611] [General]
Dear all, 
Currently I'm running the 5MW_OC4Jckt_DLL_WTurb_WavesIrr_MGrowth reg-test to simulate the OWT operational loads on the Tp parts
However with the simulations of the openfast software, it often show the warnings/ errors as the following pic.

![image](https://github.com/OpenFAST/openfast/assets/93511835/f9392571-9f58-4e7c-a2c4-03f2f7d66948)

Any suggestion or previous solutions that I might be looking for the solve the problems?

Best regard, Jack

**Comment by jjonkman** [comment]
Dear @HiHabi,

I don't see that your screen capture shows any errors, and the warnings are standard and not in need of "solving".  You will get the warning about unsteady aerodynamics from AeroDyn if you enable the unsteady airfoil aerodynamics model with constant polar data.  You will get the warning about complex eigenvalue from SubDyn if the structural model is nearly ill conditioned.

Best regards,

**Reply by HiHabi** [reply]
Dear @jjonkman, 

Thanks for you kind reply, the result displayed the openfast terminated normorlly right after the calculatin inside the terminals, does that means the OWT simulations workout properly?
Another question is, how can I recieved the static loads the OWT structure for TP parts with the parameters of "IntfFXzz, IntfMXss" and so inside the openfast desiging program as I'll need to at the initial situations in abaqus FE model previous after appling the environmental load effects results which I'd finished previously?

Thanks for your time and looking forward for your reply
Best regards, 
Jack

**Reply by jjonkman** [reply]
Dear @HiHabi,

Yes, I agree that the simulation worked properly.

I'm not sure I understand your other question though; please clarify.

Best regards,
